src/main/java/com/danielagapov/spawn/Services/User/UserService.java

```diff
@@ -70,4 +74,13 @@ public UserDTO replaceUser(UserDTO newUser, UUID id) {
             return UserMapper.toDTO(userEntity);
         });
     }
+
+    public FriendRequests saveFriendRequest(FriendRequests friendRequest) {
+        try {
+            friendRequestsRepository.save(friendRequest);
+            return friendRequest;
```

Comment:
Once you make the entity -> DTO change, please save the entity to the database, and return the DTO to the controller (for the client [mobile]).